### ISSUE TITLE
Unable to link to the subsections

### ISSUE THREAD

**Issue by mahajangaurav** [MicrosoftDocs/OfficeDocs-MicrosoftSearch#49]
Hi! I am writing about the various search features and would really like to link to the subsections described in the tree navigation:
![image](https://user-images.githubusercontent.com/10227531/74662310-ad591800-5167-11ea-9aa4-dfb99c540e77.png)

Since each of these sections expand to details various aspects within that knowledge area, it will be great if somehow each of these sections could have a landing page that others could link to. 

So basically separately be able to link to "Provide answers" and the underlying links; "Connectors" and the underlying links, etc.

Any chance that can be done? 
Thank you for all the good work!

**Comment by jeffkizn** [comment]
Thanks for the feedback. 
Those top level items are there to expand the navigation tree and do not contain any content. Thus, they do not have explicit URLs. I suggest that when you want to link to that area, you link to the top page under the section heading. That page will generally be an overview of the section.

**Comment by mahajangaurav** [comment]
Hi @jeffkizn Yes, that would have worked if the top page was an overview but it's not. It also does not have links to the underlying sections/ pages and hence the challenge. May we we can update the top page to include links to the underlying sections/ pages? Thanks!

**Comment by jeffkizn** [comment]
On each of these major sections, the first page is an overview of the feature set in that section. We do not yet have one in the Customize and Configure section.

![TOC](https://user-images.githubusercontent.com/47789222/74865057-67d25180-5305-11ea-8239-6830b39cf575.JPG)

When it comes to navigation off those pages, we'll take your feedback and see if it makes sense to include links off the overview pages. If so, we'll do so during future updates.

Thanks, again, for your feedback.
